aws-s3-private-bucket/variables.tf

```diff
@@ -62,3 +62,9 @@ variable public_access_block {
   type    = bool
   default = true
 }
+
+variable canonical_user_id_and_permissions {
```

Comment:
Can we rename to to just `grants`. The current naming does not seem to indicate what these users and permissions are used for.

Reply:
sure! the fields for `canonical_user_id` and `permissions` already indicate the information of each value, using `grants` looks good to me. (updated the summary)